ngraph/core/src/op/lstm_cell.cpp

```diff
@@ -16,8 +16,8 @@
 using namespace std;
 using namespace ngraph;
 
-constexpr NodeTypeInfo op::v4::LSTMCell::type_info;
-constexpr NodeTypeInfo op::v0::LSTMCell::type_info;
+NGRAPH_RTTI_DEFINITION(op::v0::LSTMCell, "LSTMCell", 0);
```

Comment:
Please add parent classes to all classes which are not based from Op.

Reply:
done